src/Sulu/Bundle/AdminBundle/Resources/js/views/Form/Form.js

```diff
@@ -396,17 +396,34 @@ class Form extends React.Component<Props> {
                 if (error.code === HAS_CHANGED_ERROR_CODE) {
                     this.showHasChangedWarning = true;
                     this.postponedSaveOptions = options;
+
                     return;
                 }
 
-                this.errors.push(translate('sulu_admin.form_save_server_error'));
+                this.handleSaveError(error);
             }));
     };
 
-    handleError = () => {
+    handleFormError = () => {
         this.errors.push(translate('sulu_admin.form_contains_invalid_values'));
     };
 
+    handleSaveError = (error: Object) => {
+        const {
+            route: {
+                options: {
+                    errorCodeMessages = {},
+                },
+            },
+        } = this.props.router;
+
+        const errorMessage = (error.code && errorCodeMessages[error.code])
```

Comment:
I'm not sure if this error.code is the exception code under production environment because there are 2 different serializer of exception for dev and prod in fos rest and under prod I think `code` will just return the `statusCode` (400, 500) and no error code of the exception. I don't think we will get around that we are creating a ExceptionInterface which will add somekind of `errorCode` additonal maybe `errorMessage` create our own exception serializer which I think already exist for develop.

https://github.com/sulu/sulu/issues/5336

Reply:
It should be available in `prod`; I get the following response on `sulu.rocks`:
```
{"code":1103,"message":"Conflict"}
```

Reply:
You are correct seems like this was fixed also the https://sulu.rocks/admin/api/media does work now as expected under prod. I mean code is 0 but its not longer the status code.